.github/workflows/translations.yml

```diff
@@ -5,16 +5,13 @@ on:
   schedule:
     # runs at the start of each month (UTC)
     - cron: '0 0 1 * *'
-  pull_request:
-    paths:
-      - '.github/workflows/translations.yml'
 
 jobs:
   translations:
-    # Do not run the scheduled workflow on forks, and ensure the action runs only from master branch but still from any PR
-    if: (github.event != 'schedule' || github.repository_owner == 'Cockatrice') && (github.ref_name == 'master' || github.event_name == 'pull_request')
+    # Do not run the scheduled workflow on forks
+    if: (github.event != 'schedule' || github.repository_owner == 'Cockatrice')
```

Comment:
The idea to only run it from master branch was to ensure there are no manual dispatch runs against any other branch by fault that could lead to unexpected results and weird PR's based on false or outdated strings.

Reply:
I don't think that's a concern, the dispatch selects the master branch as the default and there is no reason to lock it away